packages/core/src/publicCredential/PublicCredential.ts

```diff
@@ -80,11 +80,11 @@ function verifyClaimsStructure(input: IAssetClaim | PartialAssetClaim): void {
 
 // Used internally only when building the [[IPublicCredentialInput]].
 function verifyDataStructure(input: IPublicCredentialInput): void {
-  // Taken from `Credential.verifyDataStructure()`
-  if (!('claims' in input)) {
+  // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
```

Comment:
Right, I guess I remembered this guy when saying that TS is more accepting of the form `'xxx' in yyy

Reply:
It was returning `true` even when I did not pass it in the object. I tested that in the unit tests, and I do not know why. And also the warning is only shown for the `claims` property, somehow.

Reply:
https://github.com/KILTprotocol/sdk-js/pull/677/commits/f503afa1216bf43518598267e499a0f61f49019d